fe/src/main/java/org/apache/doris/http/rest/TableQueryPlanAction.java

```diff
@@ -116,6 +120,7 @@ protected void executeWithoutPassword(ActionAuthorizationInfo authInfo, BaseRequ
             if (Strings.isNullOrEmpty(sql)) {
                 throw new DorisHttpException(HttpResponseStatus.BAD_REQUEST, "POST body must contains [sql] root object");
             }
+            LOG.info("SQL syntax [{}]", sql);
```

Comment:
You should add more information to this log. Such as client information, user, passwd...